com.smallstep.Smallstep.yml

```diff
@@ -0,0 +1,100 @@
+id: com.smallstep.Smallstep
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: smallstep
+rename-icon: smallstep
+
+finish-args:
```

Comment:
I removed several things here to minimize the capabilities down to just what we need, to the best of my knowledge. Unfortunately, since need to access to dbus, the flatpak is always going to be labeled "unsafe".

Reply:
Notably, I removed all the bits related to Gnome/KDE proxy stuff, as this is all handled through `xdg-desktop-portal` in modern runtimes.